openqaoa/problems/helper_functions.py

```diff
@@ -28,6 +28,24 @@ def convert2serialize(obj):
     else:
         return obj
 
+def convert2serialize_complex(obj):
```

Comment:
can you add some docstring to help explain what this function is needed for?

Reply:
Done